daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -388,14 +389,15 @@ private[speedy] case class PartialTransaction(
       val auth = Authorize(context.info.authorizers)
       val nid = NodeId(nextNodeIdx)
       val node = Node.Fetch(
-        coid = coid,
-        templateId = contract.templateId,
-        actingParties = actingParties,
-        signatories = contract.signatories,
-        stakeholders = contract.stakeholders,
-        keyOpt = contract.keyOpt.map(_.globalKeyWithMaintainers),
-        byKey = normByKey(version, byKey),
-        version = version,
+        coid,
+        contract.packageName,
+        contract.templateId,
+        actingParties,
+        contract.signatories,
+        contract.stakeholders,
+        contract.keyOpt.map(_.globalKeyWithMaintainers),
+        normByKey(version, byKey),
+        version,
```

Comment:
I'd prefer to have the named arguments, as before

Reply:
Very good point.